src/form/input/Input.tsx

```diff
@@ -97,12 +97,12 @@ function Input(props: InputProps) {
   const inputContainerClassName = classNames(
     "input-container",
     customClassName,
-    `input-container--type-${type}`
+    `input-container--type-${type}`,
+    {
+      "input-container--is-disabled": isDisabled,
+      "input-container--has-error": hasError
+    }
   );
-  const inputClassName = classNames("input", {
-    "input--is-disabled": isDisabled,
-    "input--has-error": hasError
-  });
```

Comment:
Let's make sure we explain this in the release notes as this is a breaking change